### PR TITLE
fix booking tour and comment tour

### PR DESCRIPTION
**Pull request by huynhvu220796**
ready

### CODE REVIEW

**Comment by huynhvu220796** [comment]
ready

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -89,7 +87,7 @@
                 </div>
             @endforeach
             <div class="col-md-12">
-                {{ $tours->links() }}
+                {{ $tours->links("pagination::bootstrap-4") }}
```
Comment (trangkarol) [inline resources/views/home.blade.php]: Dùng dấu `' '` thay cho dấu `" "`

**Comment by trangkarol** [comment]
commented

**Comment by trangkarol** [comment]
commented

**Comment by trangkarol** [comment]
commented

**Comment by huynhvu220796** [comment]
ready

**Pull request merged**